tests/vm/tmisc_vm.nim

```diff
@@ -19,6 +19,7 @@ foo4
 (a: 0, b: 0)
 '''
 """
+import std/sets
```

Comment:
a dangerous issue
see https://github.com/nim-lang/Nim/issues/16707

Reply:
let's do https://github.com/nim-lang/Nim/pull/16698#issuecomment-759765331 instead

Reply:
PR was merged, you can undo this change now

Reply:
not necessary